.gitignore

```diff
@@ -15,6 +15,7 @@ node_modules
 config.js
 .env
 .env.development
+docker-compose.yml
```

Comment:
What is the rationale for not committing this and having a compose template? Ideally, a developer can clone the repo and everything (mostly) "just works" by running `docker-compose up` but I might be missing something.

Reply:
The path where I store the test data and backups is not the usual path for mongo. In the docker-compose file the developer has to add the path to their data that will seed the mongo service. By copying the template they have everything and just update the path.

Reply:
@ngmaloney [dce03ce](https://github.com/AMP-SCZ/dpdash/pull/367/commits/dce03cefbd6186c4d70ddf944c5497013bec23ab)